public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js

```diff
@@ -71,14 +65,14 @@ dateFormats = [
 
   updateDateRange = (dateRangeRequired) => {
     this.setState({
-          dateRangeRequired: dateRangeRequired
-      });
-    };
+      dateRangeRequired: dateRangeRequired
+    });
```

Comment:
Yep, here the content of the `toDate` field should probably be emptied when `!dateRangeRequired`. For another PR.